pkg/asset/machines/libvirt/machines.go

```diff
@@ -63,7 +63,7 @@ func provider(clusterID string, networkInterfaceAddress string, platform *libvir
 			APIVersion: "libvirtproviderconfig.openshift.io/v1beta1",
 			Kind:       "LibvirtMachineProviderConfig",
 		},
-		DomainMemory: 8192,
+		DomainMemory: 16384,
```

Comment:
@Prashanth684 we are using https://cloud.google.com/compute/docs/general-purpose-machines#n1_machines (n1-standard-16) and if we are increased the resource that much it means we are not able to accommodate cluster in this machine (I think standard e2e-libvirt runs 3master-3worker mode).

Reply:
hmm..ok yeah n1-standard-15 has only 60G..so should we reduce worker nodes to 2 and keep it at 8192 and increase master memory to 12G then ?

Reply:
@Prashanth684 I will take a look tomorrow how it all link together and update you here, first we need to manual create a gcp instance (same resource) and try to start the cluster and see if it able to execute test as expected.

Reply:
Did you get a chance to take a look at this?

Reply:
i did kick off a libvirt e2e and as I am monitoring, i see that Praveen is right. the standard machines have 64G memory which is not sufficient for this change. But even otherwise, i think we need to look into bumping the gcp instance type to something beefier because the libvirt master defaults need to be  16G anyway for consistent CI runs

Reply:
@Prashanth684 this should be accomplished in [#5841](https://github.com/openshift/installer/pull/5841)